frame/db/src/lib.rs

```diff
@@ -240,7 +225,7 @@ pub mod pallet {
         #[pallet::weight(10000)]
         pub fn submit_result(origin: OriginFor<T>, data: InputPayload) -> DispatchResult {
             ensure_signed(origin)?;
-            Self::deposit_event(Event::SQLResult(data.data));
+            Self::deposit_event(Event::GeneralResultEvent(data.data));
```

Comment:
where are the msg and status and req_id?

Reply:
status and req_id have been encoded into data